### ISSUE TITLE
gradient mixins broken?

### ISSUE THREAD

**Issue by beriahthebee** [Dogfalo/materialize#983]
Has anyone ever tried to use the gradient mixins?

It seems the gradients rely on mixin overloading .....

```sass
@mixin linear-gradient-top($default,$color1,$stop1,$color2,$stop2)
@mixin linear-gradient-top($default,$color1,$stop1,$color2,$stop2,$color3,$stop3)
@mixin linear-gradient-top($default,$color1,$stop1,$color2,$stop2,$color3,$stop3,$color4,$stop4)
```

If you try to use one of these the compiler breaks. I'm using Prepros https://prepros.io/ and at first I thought it maybe was that, but after reading all I could into SASS mixins it seems that mixins do not support overloading.

I am using the latest and greatest source.

**Comment by beriahthebee** [comment]
Oh and just so you can see the example error output

If i use the mixin like this ...

```sass
.cool-gradient {
    @include linear-gradient-top(blue, red, 0%, blue, 100%);
}
```

I get the error....

```
/sass/components/prefixer:258: required parameter $color3 is missing in call to mixin linear-gradient-top

 Try Disabling LibSass and use slow Ruby Sass from file options.
\assets\sass\app.scss
```

**Comment by Dogfalo** [comment]
Prepros is probably using an outdated LibSass, just use Ruby Sass as it says

On Mon, Mar 23, 2015 at 5:29 PM, Beriah Dutcher <notifications@github.com>
wrote:

> Oh and just so you can see the example error output
>
> If i use the mixin like this ...
>
> .cool-gradient {
>     @include linear-gradient-top(blue, red, 0%, blue, 100%);
> }
>
> I get the error....
>
> /sass/components/prefixer:258: required parameter $color3 is missing in call to mixin linear-gradient-top
>
>  Try Disabling LibSass and use slow Ruby Sass from file options.
> \assets\sass\app.scss
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Dogfalo/materialize/issues/983#issuecomment-85205814>.
>



-- 
Doggy sends his greetings from Mars.

**Comment by beriahthebee** [comment]
Yeah I did. It still gives the same error, well similar error. 

```
Error: Mixin linear-gradient-top is missing argument $color3.
        on line 84 of app.scss, in `linear-gradient-top'
        from line 84 of app.scss
  Use --trace for backtrace.
```

On the SASS webpage Prepros is a featured compiler, weird

**Comment by beriahthebee** [comment]
I also tried to install Mixture and use it to compile. I got the same error.

**Comment by Dogfalo** [comment]
Yea, I looked into it and overloading will not work. You will have to change the names of the mixins.
Fixed in e68b4b4dc58c08a1b14bc255775a12cf4423c9a6
